2021-05-20 03:35:39,808:DEBUG:certbot._internal.main:certbot version: 1.15.0
2021-05-20 03:35:39,808:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2021-05-20 03:35:39,808:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/usr/share/nginx/html/letsencrypt', '--email', 'robotixclub@nitrr.ac.in', '--agree-tos', '--no-eff-email', '-d', 'robotix.nitrr.ac.in']
2021-05-20 03:35:39,808:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2021-05-20 03:35:39,843:DEBUG:certbot._internal.log:Root logging level set at 20
2021-05-20 03:35:39,843:INFO:certbot._internal.log:Saving debug log to /var/log/letsencrypt/letsencrypt.log
2021-05-20 03:35:39,844:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2021-05-20 03:35:39,848:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Place files in webroot directory
Interfaces: IAuthenticator, IPlugin
Entry point: webroot = certbot._internal.plugins.webroot:Authenticator
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7fbef6fca220>
Prep: True
2021-05-20 03:35:39,848:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7fbef6fca220> and installer None
2021-05-20 03:35:39,848:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2021-05-20 03:35:39,866:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/124178225', new_authzr_uri=None, terms_of_service=None), 4ccf1848dab134df4a9a7b01a72cbb83, Meta(creation_dt=datetime.datetime(2021, 5, 19, 18, 39, 12, tzinfo=<UTC>), creation_host='30d663a1f659', register_to_eff=None))>
2021-05-20 03:35:39,866:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2021-05-20 03:35:39,868:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2021-05-20 03:35:40,862:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 658
2021-05-20 03:35:40,862:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:35:40 GMT
Content-Type: application/json
Content-Length: 658
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert",
  "sI1ZiwfXnRM": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
}
2021-05-20 03:35:40,863:DEBUG:certbot.display.util:Notifying user: Requesting a certificate for robotix.nitrr.ac.in
2021-05-20 03:35:40,935:DEBUG:certbot.crypto_util:Generating RSA key (2048 bits): /etc/letsencrypt/keys/0025_key-certbot.pem
2021-05-20 03:35:40,942:DEBUG:certbot.crypto_util:Creating CSR: /etc/letsencrypt/csr/0025_csr-certbot.pem
2021-05-20 03:35:40,943:DEBUG:acme.client:Requesting fresh nonce
2021-05-20 03:35:40,943:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2021-05-20 03:35:41,190:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2021-05-20 03:35:41,191:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:35:41 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0003U-n-bU9F6fhI8jc79vHn49EB-C_txmCM65zeYgUZZVA
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2021-05-20 03:35:41,191:DEBUG:acme.client:Storing nonce: 0003U-n-bU9F6fhI8jc79vHn49EB-C_txmCM65zeYgUZZVA
2021-05-20 03:35:41,191:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "robotix.nitrr.ac.in"\n    }\n  ]\n}'
2021-05-20 03:35:41,193:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDNVLW4tYlU5RjZmaEk4amM3OXZIbjQ5RUItQ190eG1DTTY1emVZZ1VaWlZBIiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9uZXctb3JkZXIifQ",
  "signature": "WIrwcz5FIv6sJ0hiuqQWm6GWDUXqSwM1Iwv0xPoR92Fnc1TvAQbFofGkjFRsJfNjM5KfOitJhS9DD_twouEqKy6Iy72fBp2Uew5MxIlAMsFcU25VgjLBJvhi3mNjICh4YG6EAT7zoqNMmOBsmjuusSR0vSkSvUfbgnVeQGD1LEQrGE2y1HAK5kZjjxhIlCrSWXZXClsb5K4sEQ1FvviDhB1biP0zjjhyuPZjmcDN32Z5IGaYBZXoni2cB0jgTNsXP4hV4Aj1FMd5NzO5xofbmNwE5vS6xtcnY13vucaytfNQikoBIFJq-V9Ox3Q6VnIkz21oTJb3eqMHP29v02rdNw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInJvYm90aXgubml0cnIuYWMuaW4iCiAgICB9CiAgXQp9"
}
2021-05-20 03:35:41,619:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 341
2021-05-20 03:35:41,619:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Thu, 20 May 2021 03:35:41 GMT
Content-Type: application/json
Content-Length: 341
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/124178225/9821046048
Replay-Nonce: 0003XGA30oioEe-12hiT-8TqwtuAXFdiDbknWip_NOu6vDg
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2021-05-27T03:35:41Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "robotix.nitrr.ac.in"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275043815"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/124178225/9821046048"
}
2021-05-20 03:35:41,619:DEBUG:acme.client:Storing nonce: 0003XGA30oioEe-12hiT-8TqwtuAXFdiDbknWip_NOu6vDg
2021-05-20 03:35:41,620:DEBUG:acme.client:JWS payload:
b''
2021-05-20 03:35:41,621:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275043815:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDNYR0EzMG9pb0VlLTEyaGlULThUcXd0dUFYRmRpRGJrbldpcF9OT3U2dkRnIiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My8xMzI3NTA0MzgxNSJ9",
  "signature": "Nnew3QMd-3MgnYVXUSheCEveY1lQZZGvWRNhHvoIC5nzqGzxRDnDycW3J-nLAA-5E44CclxISoSmZkKLCNxXiewwAbu_9VGPdmZOdpbKNPCqEizZbG0T1Lu2DpVquvQ16Bl30n3W5WC2MJQZeZZpu2-RcopxhrvezEwzrd_LfUaoAgHlLLDy_pky7RNDjwtYiRrznsjHNszXvmy3RmB9xPfSr3BuT34agkweTeI7ZDlszIHRJHnu3JIkJk-0JOa0DDa-k6Dkw2MvmQvkyYFuwkE9GPKTo26intKE3OigHqhsJFeLhgdmkozwm1l91vNSkkQR1SF4ZyUFMvqjUZ47Cw",
  "payload": ""
}
2021-05-20 03:35:41,894:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13275043815 HTTP/1.1" 200 800
2021-05-20 03:35:41,895:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:35:41 GMT
Content-Type: application/json
Content-Length: 800
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 00033WkfShzhPVAy47iZlS3Wa_I02H8ZVFaD-kpJdHwohNE
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "robotix.nitrr.ac.in"
  },
  "status": "pending",
  "expires": "2021-05-27T03:35:41Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275043815/Fbrp6w",
      "token": "34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275043815/Taoolg",
      "token": "34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275043815/EioNgQ",
      "token": "34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM"
    }
  ]
}
2021-05-20 03:35:41,895:DEBUG:acme.client:Storing nonce: 00033WkfShzhPVAy47iZlS3Wa_I02H8ZVFaD-kpJdHwohNE
2021-05-20 03:35:41,896:INFO:certbot._internal.auth_handler:Performing the following challenges:
2021-05-20 03:35:41,896:INFO:certbot._internal.auth_handler:http-01 challenge for robotix.nitrr.ac.in
2021-05-20 03:35:41,897:INFO:certbot._internal.plugins.webroot:Using the webroot path /usr/share/nginx/html/letsencrypt for all unmatched domains.
2021-05-20 03:35:41,898:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge
2021-05-20 03:35:41,899:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM
2021-05-20 03:35:41,899:INFO:certbot._internal.auth_handler:Waiting for verification...
2021-05-20 03:35:41,901:DEBUG:acme.client:JWS payload:
b'{}'
2021-05-20 03:35:41,902:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275043815/Fbrp6w:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDMzV2tmU2h6aFBWQXk0N2labFMzV2FfSTAySDhaVkZhRC1rcEpkSHdvaE5FIiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9jaGFsbC12My8xMzI3NTA0MzgxNS9GYnJwNncifQ",
  "signature": "viX5zkAJ0Vmmrf375-iJPesEQDY12uxGAILPo9N5AzLKDAKeL03TJbfffBLHuMagK8byF5FSvU4keDwxzJOmZTEoF3hhIxPanCIQMJaisynGs8-1crjzj9RmYeyZ5RtnQr9htjlZH4MffNtt-1N0yCx3NDTZbsm2lPdmk0jW6UsmKpu0YE3Y5ert596kbPTprHk1ZS-rSNtX_UZr_I6bzUZAoxu8zVs9KRD_jd624ogj-JoiJLX7ey1h9_wFoXDAdw_Z-koKI6XlSXy4URTF8S3q2HhLiK3rfKirnY3S0XKHdhdqTfxcg6WxEcBi9J5QcvRDl7mCOhwhxXSpQwkNrQ",
  "payload": "e30"
}
2021-05-20 03:35:42,190:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/13275043815/Fbrp6w HTTP/1.1" 200 186
2021-05-20 03:35:42,191:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:35:42 GMT
Content-Type: application/json
Content-Length: 186
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275043815>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275043815/Fbrp6w
Replay-Nonce: 0004i7SRb2Nv1PBtPan_QdM0yvBeNEGkqhXWWNAG0a8ZhA4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275043815/Fbrp6w",
  "token": "34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM"
}
2021-05-20 03:35:42,191:DEBUG:acme.client:Storing nonce: 0004i7SRb2Nv1PBtPan_QdM0yvBeNEGkqhXWWNAG0a8ZhA4
2021-05-20 03:35:43,193:DEBUG:acme.client:JWS payload:
b''
2021-05-20 03:35:43,194:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/13275043815:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTI0MTc4MjI1IiwgIm5vbmNlIjogIjAwMDRpN1NSYjJOdjFQQnRQYW5fUWRNMHl2QmVORUdrcWhYV1dOQUcwYThaaEE0IiwgInVybCI6ICJodHRwczovL2FjbWUtdjAyLmFwaS5sZXRzZW5jcnlwdC5vcmcvYWNtZS9hdXRoei12My8xMzI3NTA0MzgxNSJ9",
  "signature": "Vg3PEaxkNxfN5k0ObB1onTupaHtHPpsLvFTCc-3WEbsZsahu0ixmbpPOTA3MKEo4IhE0hr9Y3VWRZVswkqp0_IiLij-QQzaaEcW3CLFRDnXYvrpWhPfHlP2t8LkNldA4htHx40m9HHnvO-Rg9gK-l15OhfRrb3w-io_uZfg95CW5c9VzIPT_ogQjJnx2pjdWKGX2m1YAJBBGarlSmD6o_pfpUqz2KshT1Dya1RErDzjz_DBLBsdbV-GUlz4kuDlMLpiQXmmJQjhcYXanfdHE23Lpo_8nsMx9qZ6QnYMO7STDdJSszF9AvFlUkxCnFwtRPevuhh5XgTuv7wRoAJrxSg",
  "payload": ""
}
2021-05-20 03:35:43,469:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/13275043815 HTTP/1.1" 200 1308
2021-05-20 03:35:43,469:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 20 May 2021 03:35:43 GMT
Content-Type: application/json
Content-Length: 1308
Connection: keep-alive
Boulder-Requester: 124178225
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0003A76x5o4XvtdC0ZB--iWd20mIUll5hnnG9m0umzO9vr0
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "robotix.nitrr.ac.in"
  },
  "status": "invalid",
  "expires": "2021-05-27T03:35:41Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "Invalid response from http://robotix.nitrr.ac.in/.well-known/acme-challenge/34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM [3.108.0.154]: \"\u003chtml\u003e\\r\\n\u003chead\u003e\u003ctitle\u003e404 Not Found\u003c/title\u003e\u003c/head\u003e\\r\\n\u003cbody\u003e\\r\\n\u003ccenter\u003e\u003ch1\u003e404 Not Found\u003c/h1\u003e\u003c/center\u003e\\r\\n\u003chr\u003e\u003ccenter\u003enginx/1.20.0\u003c/ce\"",
        "status": 403
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/13275043815/Fbrp6w",
      "token": "34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM",
      "validationRecord": [
        {
          "url": "http://robotix.nitrr.ac.in/.well-known/acme-challenge/34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM",
          "hostname": "robotix.nitrr.ac.in",
          "port": "80",
          "addressesResolved": [
            "3.108.0.154"
          ],
          "addressUsed": "3.108.0.154"
        }
      ],
      "validated": "2021-05-20T03:35:42Z"
    }
  ]
}
2021-05-20 03:35:43,469:DEBUG:acme.client:Storing nonce: 0003A76x5o4XvtdC0ZB--iWd20mIUll5hnnG9m0umzO9vr0
2021-05-20 03:35:43,470:WARNING:certbot._internal.auth_handler:Challenge failed for domain robotix.nitrr.ac.in
2021-05-20 03:35:43,471:INFO:certbot._internal.auth_handler:http-01 challenge for robotix.nitrr.ac.in
2021-05-20 03:35:43,471:DEBUG:certbot._internal.reporter:Reporting to user: The following errors were reported by the server:

Domain: robotix.nitrr.ac.in
Type:   unauthorized
Detail: Invalid response from http://robotix.nitrr.ac.in/.well-known/acme-challenge/34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM [3.108.0.154]: "<html>\r\n<head><title>404 Not Found</title></head>\r\n<body>\r\n<center><h1>404 Not Found</h1></center>\r\n<hr><center>nginx/1.20.0</ce"

To fix these errors, please make sure that your domain name was entered correctly and the DNS A/AAAA record(s) for that domain contain(s) the right IP address.
2021-05-20 03:35:43,472:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 179, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2021-05-20 03:35:43,472:DEBUG:certbot._internal.error_handler:Calling registered functions
2021-05-20 03:35:43,472:INFO:certbot._internal.auth_handler:Cleaning up challenges
2021-05-20 03:35:43,472:DEBUG:certbot._internal.plugins.webroot:Removing /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/34tbb3FAVbl3taCKYlhJNXxlj3x6aJUX4J38l1K8PkM
2021-05-20 03:35:43,473:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2021-05-20 03:35:43,473:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 15, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1435, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1304, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 140, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 444, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 374, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 424, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 91, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 179, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2021-05-20 03:35:43,474:ERROR:certbot._internal.log:Some challenges have failed.
